source/adminguide/accounts.rst

```diff
@@ -279,17 +279,100 @@ or ApacheDS to authenticate CloudStack end-users. CloudStack will search
 the external LDAP directory tree starting at a specified base directory
 and gets user info such as first name, last name, email and username.
 
-Starting with CloudStack 4.11, an ldap connection per domain can be
-defined.
+Starting with CloudStack 4.11, an LDAP connection per domain can be
+defined. In this domain autosync per account can be configured,
+keeping the users in the domain up to date with their group membership
+in LDAP.
+
+.. Note:: A caveat with this is that ApacheDS does not yet support the
+virtual 'memberOf' attribute needed to check if a user moved to
+another account. Microsoft AD and OpenLDAP as well as OpenDJ do support
+this. It is a planned feature for ApacheDS that can be tracked in
+https://issues.apache.org/jira/browse/DIRSERVER-1844.
```

Comment:
Looks like the multiline `Note` didn't work... sphinx only put the first line in the box. :disappointed: 
It should be formatted like this: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes

Reply:
fixed